modules/pytket-honeywell/pytket/extensions/honeywell/backends/honeywell.py

```diff
@@ -336,6 +336,9 @@ def process_circuits(
             basebody["group"] = group
 
         handle_list = []
+        # set some default max-batch-cost
+        batch_exec: Union[int, str] = 1
```

Comment:
Checking how the `batch-exec` field works, it seems from this code that:
* In the first circuit in a batch, `batch-exec` is set to `1`.
* In the second and subsequent circuits in a batch, `batch-exec` is set to the job ID of the first circuit in the batch, as retrieved from Honeywell's API.

Is that correct?

If so, it might be worth adding an explanatory comment here.

Reply:
Yes that is correct, I've added some comments and documentation, thanks for the catch.